client/src/app/routes/Landing.scss

```diff
@@ -225,9 +225,26 @@
               display: flex;
               position: relative;
 
-              button:first-child {
+              .pause-button {
                 margin-right: 16px;
               }
+
+              .filters-button-wrapper {
+                  position: relative;
+                  .active-filters-counter {
```

Comment:
Rename .active-filters-counter class to .filters-button-wrapper__counter to keep in line with BEM naming convention.